smoke/bridge_neigh_suppress_test.sh

```
@@ -0,0 +1,71 @@
+	tcpdump -c1 -t -nn -i x-p1 'arp and arp[24:4] = 0x0a000063' \
+	>$tmp/tcpdump.out 2>/dev/null &
+tcpdump_pid=$!
+sleep 0.5
+
+# Send an ARP request from host-a for the suppressed IP.
+ip netns exec host-a arping -c1 -w1 -I x-p0 10.0.0.99 >$tmp/arping.out
+
+# Wait for tcpdump to finish.
+wait $tcpdump_pid 2>/dev/null || true
+
+if grep -q ARP $tmp/tcpdump.out; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)